python/ray/dag/compiled_dag_node.py

```diff
@@ -30,6 +31,10 @@
 
 MAX_BUFFER_SIZE = int(100 * 1e6)  # 100MB
 
+MAX_BUFFER_TOTAL_MEMORY = int(10 * 1e9)  # 10GB
```

Comment:
Could we make these configurable by the user by passing them as args to `experimental_compile`?